registry/remote/repository.go

```diff
@@ -1106,7 +1106,7 @@ func (s *manifestStore) deleteWithIndexing(ctx context.Context, target ocispec.D
 //
 // References:
 //   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#deleting-manifests
-//   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#deleting-manifests
+//   - https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#deleting-manifests
```

Comment:
```suggestion
```

Reply:
Oh this was a bug, thanks for catching it! Both versions of specs are supported so they are listed here.